content/en/docs/ops/integrations/jaeger/index.md

```diff
@@ -7,6 +7,10 @@ owner: istio/wg-environments-maintainers
 test: n/a
 ---
 
+{{< tip >}}
```

Comment:
This tip should be move to boilerplate since it's used in three docs

Reply:
will do this in a following PR.